src/components/deprecated-warning/deprecated-warning.tsx

```diff
@@ -1,20 +1,23 @@
 import { FC } from 'react';
 import Link from '../link/link';
 import Tip from '../tip/tip';
+import { ElementContent } from '../../types';
 
 interface DeprecatedWarningProps {
   alternativeName: string;
   alternativeLink: string;
+  children: ElementContent;
 }
 
-const DeprecatedWarning: FC<DeprecatedWarningProps> = ({ alternativeName, alternativeLink }) => (
+const DeprecatedWarning: FC<DeprecatedWarningProps> = ({ alternativeName, alternativeLink, children }) => (
   <Tip emoji="🚨" title="Deprecated component" type={Tip.types.DANGER}>
     <>
       This is a legacy component and will be deprecated in the next major version. Please consider using the
       <Link href={alternativeLink} size={Link.sizes.SMALL}>
         {alternativeName}
       </Link>
       component for your needs instead.
+      {children}
```

Comment:
I'd call it `additionalContent`. usually `children` is used as something that can replace the entire content and not just to add some parts.